save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/TestStatusComponent.kt

```diff
@@ -66,7 +66,10 @@ fun <D : Any> testStatusComponent(testResultDebugInfo: TestResultDebugInfo, tabl
             colSpan = 2
             +"Reason ("
             a {
-                href = "${window.location}/details/$testSuiteName/$pluginName/$testFilePath"
+                // Trim location until the value of number of execution, since it could contain filters at the end, like `?status=PASSED`,
+                // when user got to this page by clicking corresponding table column on history view
+                val baseLocation = window.location.toString().dropLastWhile { !it.isDigit() }
```

Comment:
I think we can safely drop substring after `?`

Reply:
I thought about it, but may be it is more safe way, independent of any changes in our API in future

Reply:
I think `?`, which is a dedicated delimiter in the general URL format, is more independent. Relying on digits in the link is in fact bound to changes of our API

Reply:
Ok, changed